src/components/clusterConfiguration/DiscoveryImageTypeControlGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Split, SplitItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import { PopoverIcon } from '../ui';
+import { RadioField } from '../ui/formik/RadioField';
+
+const GROUP_NAME = 'imageType';
+
+const DiscoveryImageTypeControlGroupLabel = ({
+  text,
+  popoverContent,
+}: {
+  text: string;
+  popoverContent: string;
+}) => (
+  <>
+    {text}
+    <PopoverIcon
+      component={'a'}
```

Comment:
Do not the `PopoverIcon` defaults serve our need even hear?

Reply:
@mareklibra no, unfortunately. The default PopoverIcon rendered the button in blue.